datafusion/proto/src/generated/mod.rs

```diff
@@ -17,7 +17,6 @@
 
 #[allow(clippy::all)]
 #[rustfmt::skip]
-#[cfg(not(docsrs))]
```

Comment:
I think this was added in  https://github.com/apache/datafusion/pull/3580 but now that the code is vendored (doesn't require running protoc at compile time) I think it should work

However, I don't really understand why it stopped working in the first place 🤔